src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -370,50 +371,83 @@ def _accumulate_energy(self, area, child, is_cep: bool):
                     if trade.buyer is child.name:
```

Comment:
In this function an above, a distinction between "nromal" trades and trades including ceps is done. AFAIS, this is not needed any more. So, IMO you can (and should) remove the self._cep_energy and self._cep_device members. Or did i miss something?

Reply:
You should ignore it, as its only used in `_accumulated_trade_energy` method which is not used anywhere anymore and I will remove it to avoid confusion.